examples/mysql/getting_started_step_3/src/bin/publish_post.rs

```diff
@@ -16,10 +16,7 @@ fn main() {
         .expect("Invalid ID");
     let connection = establish_connection();
 
-    let post: Post = posts
-        .find(id)
-        .first(&connection)
-        .unwrap_or_else(|_| panic!("Unable to find post {}", id));
+    let post: Post = posts.find(id).first(&connection).unwrap();
```

Comment:
The error message was correct in that case, or do I miss something?

Reply:
uh, I might have been too quick there. let me look at it again.

Reply:
Yeah, sorry! Good catch! :flushed: